frontend/src/components/common/Table.tsx

```diff
@@ -42,10 +43,8 @@ export const Table = ({ headers, rows }: TableProps): React.ReactElement => {
               key={rowIndex}
               backgroundColor={rowIndex % 2 === 0 ? "blue.50" : "grey.50"}
             >
-              {row.values.map((value, cellIndex) => (
-                <Td key={value} fontWeight={cellIndex === 0 ? "bold" : ""}>
-                  {value}
-                </Td>
+              {row.values.map((value) => (
+                <Td key={rowIndex}>{value}</Td>
```

Comment:
Can we leave logic in for bolding here? Based on the designs that we have, I think the first string column of a table is still always bolded and that logic should probably be reflected in this `Table` component.

One option is that you could check for the first string column and bold it and then you can remove the additional bold attribute from both `AdminUserTable` and `AssessmentsTable`.